src/pca_gophish.yml

```diff
@@ -21,25 +21,29 @@
 
     - name: Create mount point for GoPhish persistent data volume
       file:
+        mode: 0644
```

Comment:
This mode should be `0755` since it is a directory.

Reply:
Thanks! Fixed in https://github.com/cisagov/pca-gophish-composition-packer/pull/22/commits/dfa4c51232c2892092808b148c837e0d082cef1d.